cpp/oneapi/dal/algo/svm/backend/cpu/infer_kernel_predict_fast.cpp

```diff
@@ -0,0 +1,124 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <daal/src/algorithms/svm/svm_predict_kernel.h>
+
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+#include "oneapi/dal/algo/svm/backend/cpu/infer_kernel.hpp"
+
+namespace oneapi::dal::svm::backend {
+
+using std::int64_t;
+using dal::backend::context_cpu;
+
+namespace daal_svm = daal::algorithms::svm;
+namespace daal_kernel_function = daal::algorithms::kernel_function;
+namespace interop  = dal::backend::interop;
+
+template <typename Float, daal::CpuType Cpu>
+using daal_svm_predict_kernel_t = daal_svm::prediction::internal::SVMPredictImpl<daal_svm::prediction::defaultDense, Float, Cpu>;
+
+struct daal_svm_model: public daal_svm::Model {
+    daal_svm_model() = default;
+    virtual ~daal_svm_model() {}
+
+    auto& set_support_vectors(daal::data_management::NumericTablePtr support_vectors) {
+        _SV = support_vectors;
+        return *this;
+    }
+
+    auto& set_coefficients(daal::data_management::NumericTablePtr coefficients) {
+        _SVCoeff = coefficients;
+        return *this;
+    }
+
+    auto& set_bias(double bias) {
+        _bias = bias;
+        return *this;
+    }
+};
```

Comment:
We are trying not to propagate old code to the new oneDAL library in every implementation file. For oneDAL-DAAL integration, we have an `interop` module: https://github.com/oneapi-src/oneDAL/tree/master/cpp/oneapi/dal/backend/interop.
I'd suggest creating converters/adapters on DAAL in this module. It would be easier to find such conversions, rather than they will be distributed across all the library. The only exception, in my opinion, should be `call_daal_kernel`, but it can be also moved to the interop module.

Reply:
Thanks for the comment. In this case, there is no conversion. There is only an extension of the existing functionality of the old daal. Is it worth moving to `interop`?

Reply:
Yes. All staff that interacts with DAAL shall be located inside `interop` module.

Reply:
Applied